var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -193,6 +193,9 @@ def libs(self):
     # 1.84.0 dropped support for 98/03
     conflicts("cxxstd=98", when="@1.84.0:")
 
+    # Autotools packages are not available on Windows
```

Comment:
Yes, concretization fails when you add `+icu` to boost (after I took out the conflict) due to autotools. Should I take out the conflict then since it is handled elsewhere?

Reply:
Yes - thanks!